apps/deploy_monitoring/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "deploy-monitoring"
+version = "1.2.0"
+authors = ["Adrian Nagy <adrian.nagy@viablesystems.io>"]
+edition = "2018"
+default-run = "deploy-monitoring"
+
+[dependencies]
+async-trait = "0.1"
+clap = "2.33"
+chrono = "0.4"
+failure = "0.1"
+fs2 = "0.4"
+fs_extra = "1.2"
+itertools = "0.10"
+getset = "0.1"
+libflate = "1"
+merge = "0.1"
+http = "0.2.2"
+percentage = "0.1"
+reqwest = { version = "0.11", features = ["json"] }
+serde = { version = "1.0", features = ["derive", "rc"] }
+serde_json = "1.0"
+shiplift = { git = "https://github.com/simplestaking/shiplift.git", branch = "master" }
+slog = { version = "2.5", features = ["nested-values"] }
+slog-async = "2.5"
+slog-term = "2.6"
+sysinfo = "0.16"
+tokio = { version = "1.2", features = ["full"] }
+wait-timeout = "0.2"
+warp = "0.3"
+shell = { path = "../../shell" }
```

Comment:
@adonagy 
please add here `TODO: TE-499 remove shell dependency, and move stats/memory somewhere`